packages/utils/copy-utils.ts

```diff
@@ -8,18 +8,18 @@ const path = require("path");
  * @param {string} templateDir Absolute path to template directory
  * @returns {Function} A curried function that takes a file path and copies it
  */
-const generatorCopy = (
-	generator,
-	templateDir
-) => /** @param {string} filePath */ filePath => {
-	const sourceParts = templateDir.split(path.delimiter);
+export const generatorCopy = (
+	generator: any,
+	templateDir: string,
+): (filePath: string) => void => /** @param {string} filePath */ (filePath: string): void => {
```

Comment:
I think this comment is not longer needed

Reply:
same